docs/training-debug/index.txt

```diff
@@ -1,343 +1,395 @@
+   <step4>`.
 
--  A custom Docker image (if required) is set in the experiment config.
+.. _step7:
 
--  Any ``pip install`` or ``apt install`` commands needed in the interactive environment are either
-   built into a custom Docker image or written into a file called ``startup-hook.sh`` in the root of
-   the model definition directory. See :ref:`startup-hooks` for more details.
+****************************************************************************
+ Step 7 - Verify that cluster test mode works with slots_per_trial set to 1
+****************************************************************************
 
--  Any custom bind mounts that were required in the interactive environment are also specified in
-   the experiment config.
+This step is similar to :ref:`Step 6 <step6>`, except instead of launching the command from an
+interactive environment, it is submitted to the cluster and managed by Determined.
 
--  Environment variables are set properly in the experiment config.
+#. Apply customizations:
 
-If no missing customizations are to blame, there are still several new layers introduced with a
-cluster-managed experiment that would not cause issues with local training mode:
+   If you customized your command environment in testing :ref:`Step 3 <step3>`, :ref:`Step 4
+   <step4>`, or :ref:`Step 5 <step5>`, make sure to apply the same customizations in your experiment
+   configuration file.
 
--  The ``checkpoint_storage`` settings are used for cluster-managed training. If
-   ``checkpoint_storage`` was configured in neither the experiment config nor the master config, you
-   will see an error message during experiment config validation, before the experiment or any
-   trials are created. To correct it, simply provide a ``checkpoint_storage`` configuration in one
-   of those locations (:ref:`master-configuration` or :ref:`experiment-configuration`).
+#. Set ``resources.slots_per_trial``:
 
--  The configured ``checkpoint_storage`` settings are validated before training starts for an
-   experiment on the cluster. If you get a message saying ``Checkpoint storage validation failed``,
-   please review the correctness of the values in your ``checkpoint_storage`` settings.
+   Confirm that your experiment config does not specify ``resources.slots_per_trial`` or that it is
+   set to ``1``. For example:
 
--  The experiment config is fully validated for cluster-managed experiments, more strictly than it
-   is for ``--local --test`` mode. If you get errors related to ``unmarshaling JSON`` when trying to
-   submit the experiment to the cluster, that is an indication that the experiment config has
-   errors. Please review the :ref:`experiment configuration <experiment-configuration>`.
+   .. code:: yaml
 
-Again, if you are unable to identify the root cause of the issue yourself, please do not hesitate to
-contact Determined through our `community support
+      resources:
+      slots_per_trial: 1
```

Comment:
the `slots_per_trial` line needs an indent to be valid yaml

Reply:
fixed.